src/main/java/picard/arrays/illumina/InfiniumEGTFile.java

```diff
@@ -98,21 +88,21 @@ private void readFileData() throws IOException {
         initializeArrays(numCodes);
 
         for (int i = 0; i < numCodes; i++) {
-            nAA[i] = parseInt();
-            nAB[i] = parseInt();
-            nBB[i] = parseInt();
-            devRAA[i] = parseFloat();
-            devRAB[i] = parseFloat();
-            devRBB[i] = parseFloat();
-            meanRAA[i] = parseFloat();
-            meanRAB[i] = parseFloat();
-            meanRBB[i] = parseFloat();
-            devThetaAA[i] = parseFloat();
-            devThetaAB[i] = parseFloat();
-            devThetaBB[i] = parseFloat();
-            meanThetaAA[i] = parseFloat();
-            meanThetaAB[i] = parseFloat();
-            meanThetaBB[i] = parseFloat();
+            for (InfiniumVcfFields.GENOTYPE_VALUES gtValue : InfiniumVcfFields.GENOTYPE_VALUES.values()) {
```

Comment:
would it make sense to define two functions `parseInts` and `parseFloats` to avoid this boilerplate?

Reply:
then each 3 lines would become
```
parseInts(devR[i]) 
```

and parseInts would be
```java
void parseInts(int[] array){
   for(int i=0;i<array.length;i++){
      array[i]=parseInt();
   }
}
```

Reply:
Done.  Thanks.